### ISSUE TITLE
Animation doesn't appear until animation data is fully loaded 

### ISSUE THREAD

**Issue by mikob** [airbnb/lottie-web#1272]
My lottie animation (seen here: https://www.lipsurf.com) does not appear until after the json animation data is *fully* loaded (/assets/anim/lipsurf-dude-2.json). The animation data is 1.3mb (250kB gzipped) and even on a fast connection there is a visible delay before the initial frame shows. 

Here's the settings I'm using:
```js
    var animation = bodymovin.loadAnimation({
      container: this.$refs.lipsurfDudeAnim,
      path: "/assets/anim/lipsurf-dude-2.json",
      renderer: "svg",
      loop: true,
      autoplay: true,
      rendererSettings: {
        progressiveLoad: true
      }
    });
```

I know it's not showing anything until after the json is fully loaded, because I've slowed my connection down using the chrome dev tools and can see it doesn't show the initial frame until after the json request finishes.

Am I misunderstanding the `progressiveLoad` option or is there something else I'm doing wrong? Ideally I would like to show an initial frame while the animation loads, then play the animation. If there's an existing workaround for this, it would be a huge help!

Thank you!

**Comment by bodymovin** [comment]
Hi, `progressiveLoad` is meant to add elements to the dom progressively instead of loading all at once, but the json needs to be fully loaded.
Depending on the way the animation layers are organized, you can split the animation in multiple segments (multiple jsons) while exporting.
If that doesn't help, you can save an "svg screenshot" of the first frame and display it until the animation is ready.

**Comment by mikob** [comment]
@bodymovin thanks! I will try the svg screenshot method and report back here. If I find a clean way to do it I'll update the docs.

**Comment by mikob** [comment]
I ended up doing an "svg screenshot". I had to add the little hack below to clear the svg's container once the data is loaded. 

@bodymovin A small suggestion, something like the `clearCanvas` property could be useful for svgs to, maybe if it was just named `clearBeforeLoad` and both cleared the canvas or emptied the svg container before loading the first frame.

```
		// hack to clear initial frame
		animation.addEventListener('data_ready', () => {
			// @ts-ignore
			this.$refs.lipsurfDudeAnim.removeChild(this.$refs.lipsurfDudeAnim.firstChild);
		});
```
